PC/pyconfig.h.in

```diff
@@ -102,6 +102,11 @@ WIN32 is still required for the locale module.
 /* #define Py_GIL_DISABLED 1 */
 #endif
 
+// _DEBUG implies Py_DEBUG
+#ifdef _DEBUG
+#define Py_DEBUG
+#endif
```

Comment:
This code already exists elsewhere.

Reply:
Just move the location to allow the follow-up code to the `Py_DEBUG` macro.